.github/workflows/compfuzzci_no_forks.yaml

```diff
@@ -0,0 +1,91 @@
+name: CompFuzzCI
+on:
+  pull_request:
+    branches:
+      - master
+    types: [opened, closed, synchronize, reopened]
+  issues:
```

Comment:
Why is `issues` here? Why should this trigger when someone makes an issue?

Reply:
It's for keeping track of active bugs in dafny. If the fuzzer finds the bug that has already been reported in some issue, it can ignore that bug.